test/bigtest/interactors/transactions/TransactionDetailsInteractor.js

```diff
@@ -1,13 +1,20 @@
 import {
   interactor,
   isPresent,
+  text,
 } from '@bigtest/interactor';
 import Button from '../common/Button';
 
+@interactor class Status {
+  static defaultScope = '[data-test-transaction-status]';
+  value = text('[class*=kvRoot---]');
```

Comment:
imo it's beter to add span with data-test instead of relying on `class*=kvRoot---`

Reply:
Done